plugins/octant/cmd/antrea-octant-plugin/main.go

```diff
@@ -111,97 +118,46 @@ func handleNavigation(request *service.NavigationRequest) (navigation.Navigation
 				Path:     request.GeneratePath("components/agent"),
 				IconName: "folder",
 			},
+			{
+				Title:    traceflowTitle,
+				Path:     request.GeneratePath("components/traceflow"),
+				IconName: "folder",
+			},
 		},
 		IconName: "cloud",
 	}, nil
 }
 
 // initRoutes routes for Antrea plugin.
-func initRoutes(router *service.Router) {
-	controllerCols := component.NewTableCols(versionCol, podCol, nodeCol, serviceCol, crdCol, heartbeatCol)
-	agentCols := component.NewTableCols(versionCol, podCol, nodeCol, subnetCol, bridgeCol, podNumCol, crdCol, heartbeatCol)
-
-	// Click on navigation child named Overview to display Antrea components (both Controller and Agent) information.
+func (a *octantPlugin) initRoutes(router *service.Router) {
+	// Click on navigation child named Overview to display all Antrea information.
 	router.HandleFunc("/components/overview", func(request service.Request) (component.ContentResponse, error) {
-		controllerRows := getControllerRows()
-		agentRows := getAgentRows()
-		return component.ContentResponse{
-			Title: component.TitleFromString(title),
-			Components: []component.Component{
-				component.NewTableWithRows(controllerTitle, "", controllerCols, controllerRows),
-				component.NewTableWithRows(agentTitle, "", agentCols, agentRows),
-			},
-		}, nil
+		handlers := []service.HandleFunc{
+			a.controllerHandler,
+			a.agentHandler,
+			a.traceflowHandler,
+		}
+		resp := component.NewContentResponse(component.TitleFromString(title))
+		for _, handler := range handlers {
+			newTab, err := handler(request)
+			if err != nil {
+				log.Printf("Failed to load a tab in overview page, err: %v", err)
+				return component.EmptyContentResponse, err
+			}
+			for i, c := range newTab.Components {
+				c.SetAccessor(newTab.Title[0].String() + strconv.Itoa(i))
```

Comment:
I believe this is to ensure we will show controller tab first. Could you please add some comments here?

Reply:
Comment added.